src/api/public/apidocs-new/components/schemas/reason.yaml

```diff
@@ -0,0 +1,13 @@
+type: object
+properties:
+  explain:
+    type: string
+    example: 'source change'
+  time:
+    type: string
+    example: '1617982407'
+  oldsource:
+    type: string
+    example: 'da9ebc6a12b6da45f9ead3fc2f924889'
+xml:
+  name: reason
```

Comment:
Nitpick: regarding the name of the file `reason.yaml`, I find it too generic. Even it describes an XML that contains a `<reason>` element, listing it together with all the other schemas makes it not so obvious that it refers to a build reason. Maybe is it better to change the name of the file from `reason.yaml` to `buildreason.yml`, as states in the original apidocs?

Reply:
I had the same feeling, yes. But I would have suggested `build_reason.yaml`. However, as we are still using terms like `jobstatus` I'm fine to continue using non-separated words like `buildreason`.

Reply:
Done @eduardoj